thirdeye/thirdeye-frontend/app/pods/rootcause-aggregates-cache/service.js

```diff
@@ -90,14 +116,12 @@ export default Service.extend({
       res = await this._extractAggregatesBatch(res, urns);
```

Comment:
Nice job on using async/await! 
Looks like some `await`s here are not needed though. We should only use `await` with an async call (ajax for example).

Reply:
Removed unnecessary awaits